frontend/pages/my-flathub.tsx

```diff
@@ -42,6 +44,25 @@ export default function Userpage({
           <div className="space-y-12">
             <UserApps variant="dev" />
 
+            {!IS_PRODUCTION && (
```

Comment:
I've started to guard cases like this with is production or is moderator for now.